src/test/java/plugins/ConfigFileProviderTest.java

```diff
@@ -0,0 +1,95 @@
+package plugins;
+
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.plugins.config_file_provider.ConfigFileProvider;
+import org.jenkinsci.test.acceptance.plugins.config_file_provider.MavenSettingsConfig;
+import org.jenkinsci.test.acceptance.plugins.config_file_provider.ServerCredentialMapping;
+import org.jenkinsci.test.acceptance.plugins.credentials.CredentialsPage;
+import org.jenkinsci.test.acceptance.plugins.credentials.ManagedCredentials;
+import org.jenkinsci.test.acceptance.plugins.credentials.UserPwdCredential;
+import org.jenkinsci.test.acceptance.po.Build;
+import org.jenkinsci.test.acceptance.po.WorkflowJob;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Tests config-file-provider plugin inside a Pipeline. 2.15.5 includes a major refactor as well as some bug fixes
```

Comment:
It does not seem to be testing any features on 2.15.5. There is no reason to require it.